src/documents/rankingDummyData.json

```
@@ -1,54 +1,54 @@
 [
     {
         "id": "",
-        "name": "平田海輝"
+        "name": "a太郎"
     },
     {
         "id": "",
-        "name": "吉本胡葉"
+        "name": "b太郎"
     },
     {
         "id": "",
-        "name": "河野太斉"
+        "name": "c太郎"
     },
     {
         "id": "",
-        "name": "林壱仁"
+        "name": "d太郎"
     },
     {
         "id": "",
-        "name": "藤本陽向"
+        "name": "e太郎"
     },
     {
         "id": "",
-        "name": "中嶋柊"
+        "name": "f太郎"
     },
     {
         "id": "",
-        "name": "中村夏都"
+        "name": "g太郎"
     },
     {
         "id": "",
-        "name": "田代睦稀"
+        "name": "h太郎"
     },
     {
         "id": "",
-        "name": "宮本奏汰"
+        "name": "i太郎"
     },
     {
         "id": "",
-        "name": "粟村翔"
+        "name": "j太郎"
     },
     {
         "id": "",
-        "name": "クボT.A.秀哉"
+        "name": "T.A太郎"
     },
     {
         "id": "",
-        "name": "高山日菜子"
+        "name": "k太郎"
     },
     {
         "id": "",
-        "name": "尾形智央"
+        "name": "l太郎"
     }
 ]
```

他のファイルはおそらく、ファイルの最後に改行を入れるように統一していたと思うので、そうした方が良さそう。

```suggestion
] 

```